src/src/com/microsoft/aad/adal/TokenCacheItem.java

```diff
@@ -66,39 +66,76 @@
     private String mFamilyClientId;
 
     /**
-     * Construct default cache item.
+     * Default constructor for cache item.
      */
-    public TokenCacheItem() {
-
+    public TokenCacheItem() {}
+    
+    TokenCacheItem(final TokenCacheItem tokenCacheItem) {
```

Comment:
Reuse the same token...

---
In reply to: [63266584](https://github.com/AzureAD/azure-activedirectory-library-for-android/pull/611#discussion_r63266584) [](ancestors = 63266584)